docs/installation/bootstrap.rst

```diff
@@ -207,6 +208,20 @@ defaults kubernetes configuration.
   configure the corresponding entries in the
   ``kubernetes.apiServer.featureGates`` mapping.
 
+  If ``dex`` is enabled, it will be used as ``oidc`` for ``kube-apiserver``
+  but you can use a `specific OpenID for kube-apiserver`_, to do so:
```

Comment:
Btw I'm not sure it really makes sense to put this in the dex section,
Because if you disable Dex there is no impact on apiserver itself, except that it will not use Dex but it's obvious. To me what we want to reflect is that oidc for apiserver is configurable and default to dex when it's deployed. 
WDYT ?

Reply:
Ah right, we only have Grafana / MetalK8s UI if we use Dex, today at least... So indeed, no need to add anything to the addons paragraph, until we manage these.